e2e-tests/development-runtime/cypress/integration/hot-reloading/error-handling/compile-error.js

```diff
@@ -0,0 +1,41 @@
+before(() => {
+  cy.exec(
+    `npm run update -- --file src/pages/error-handling/compile-error.js --restore`
+  )
+})
+
+after(() => {
+  cy.exec(
+    `npm run update -- --file src/pages/error-handling/compile-error.js --restore`
+  )
+})
+
+const errorPlaceholder = `// compile-error`
+const errorReplacement = `a b`
+
+describe(`testing error overlay and ability to automatically recover from webpack compile errors`, () => {
+  it(`displays content initially (no errors yet)`, () => {
+    cy.visit(`/error-handling/compile-error/`).waitForRouteChange()
```

Comment:
Important part in those tests that we have just single initial `cy.visit` and we don't interact with page in any way (don't manually refresh, don't click anything, we only do changes to either source code or content and assert that fixing the problem cause runtime to work again automatically - even if we do refresh browser ourselves, user doesn't have to).